samples/addons/kiali.yaml

```diff
@@ -0,0 +1,426 @@
+---
+# Source: kiali-server/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kiali
+  namespace: "kmesh-system"
+  labels:
+    helm.sh/chart: kiali-server-2.1.0
+    app: kiali
+    app.kubernetes.io/name: kiali
+    app.kubernetes.io/instance: kiali
+    version: "v2.1.0"
+    app.kubernetes.io/version: "v2.1.0"
+    app.kubernetes.io/part-of: "kiali"
+...
+---
+# Source: kiali-server/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kiali
+  namespace: "kmesh-system"
+  labels:
+    helm.sh/chart: kiali-server-2.1.0
+    app: kiali
+    app.kubernetes.io/name: kiali
+    app.kubernetes.io/instance: kiali
+    version: "v2.1.0"
+    app.kubernetes.io/version: "v2.1.0"
+    app.kubernetes.io/part-of: "kiali"
+data:
+  config.yaml: |
+    additional_display_details:
+    - annotation: kiali.io/api-spec
+      icon_annotation: kiali.io/api-type
+      title: API Documentation
+    auth:
+      openid: {}
+      openshift:
+        client_id_prefix: kiali
+      strategy: anonymous
+    clustering:
+      autodetect_secrets:
+        enabled: true
+        label: kiali.io/multiCluster=true
+      clusters: []
+    deployment:
+      additional_service_yaml: {}
+      affinity:
+        node: {}
+        pod: {}
+        pod_anti: {}
+      cluster_wide_access: true
+      configmap_annotations: {}
+      custom_envs: []
+      custom_secrets: []
+      dns:
+        config: {}
+        policy: ""
+      host_aliases: []
+      hpa:
+        api_version: autoscaling/v2
+        spec: {}
+      image_digest: ""
+      image_name: quay.io/kiali/kiali
+      image_pull_policy: IfNotPresent
+      image_pull_secrets: []
+      image_version: v2.1
```

Comment:
I see kiali has `latest` tag mirror.
Is it better to replace it with latest? It would reduce the maintenance cost of this configuration.

Reply:
I would prefer using a fixed one, we can upgrade manually

Reply:
> I see kiali has `latest` tag mirror. Is it better to replace it with latest? It would reduce the maintenance cost of this configuration.

For Kiali, I think we may need some manually upgrades. Because Kiali may adjust its logic to maintain or optimize the compatibility with Istio at any time.